cypress/fixtures/politicsArticlesResponse.json

```diff
@@ -3,142 +3,26 @@
     {
       "id": 360,
       "title": "Putin and Trump",
-      "body": "Shocking news, USA and Russia are...",
-      "created_at": "2022-01-13T10:24:38.354Z",
-      "updated_at": "2022-01-13T10:24:38.354Z"
+      "teaser": "Shocking news, USA and Russia are...",
+      "published": "2022-01-13T10:24:38.354Z"
```

Comment:
Fixture doesn't match the api response:
{"id"=>236,
    "title"=>"Covid-19 is a pandemic",
    "teaser"=>"This is my article body...",
    "published"=>"2022-01-23T16:32:50.881Z",
    "category"=>"Politics"}
Category is missing.